### ISSUE TITLE
Delly with empty BED file

### ISSUE THREAD

**Issue by fgvieira** [dellytools/delly#244]
Dear all,

running `delly` with an empty BED gives an error:
```
Exclude file is missing: [...]
```

Would it be possible to, when BED file is empty, to just ignore it and print a warning? I think it would be just a matter of tweaking these lines:
https://github.com/dellytools/delly/blob/cace738093348cb8b1c12b26a83d527c0a92c667/src/delly.h#L340-L346

thanks,

**Comment by tobiasrausch** [comment]
The BED file is not a mandatory option. If it is empty you shouldn't specify it on the command-line.

**Comment by fgvieira** [comment]
yes, but when running `delly` through a pipeline, you not always know if the BED file will be empty or not (since `delly` requires the complement).
Another way would be to allow for a BED with the regions to include (rather than exclude).

**Comment by fgvieira** [comment]
Hi @tobiasrausch,

I see that you just released a new version of `delly`, would it be possible to also have an include BED file option (like, e.g., in `manta` or other SV callers)?
It would be a great help for me. :smile: 

thanks in advance for your help.
